plyer/platforms/android/inappbrowser.py

```diff
@@ -0,0 +1,23 @@
+from jnius import cast
+from jnius import autoclass
+from plyer.facades import InAppBrowser
+
+PythonActivity = autoclass('org.renpy.android.PythonActivity')
```

Comment:
Should be probably changed to [`org.kivy.android`](https://github.com/kivy/python-for-android/blob/master/pythonforandroid/bootstraps/sdl2/build/src/org/renpy/android/PythonActivity.java#L7), but I'm not sure if there's still some kind of support for pygame or not.